types.ts

```diff
@@ -349,57 +526,129 @@ export type VariousSupportedAuthentication =
   | MultiQueryParamTokenAuthentication
   | WebBasicAuthentication;
 
+/**
+ * @ignore
+ */
 export type VariousSupportedAuthenticationTypes = $Values<Pick<VariousSupportedAuthentication, 'type'>>;
 
+/**
+ * Definition for a custom column type that you apply to any column in any Coda table.
+ * A column format tells Coda to interpret the value in a cell by executing a formula
+ * using that value, typically looking up data related to that value from a third-party API.
+ * For example, the Weather pack has a column format "Current Weather"; when applied to a column,
+ * if you type a city or address into a cell in that column, that location will be used an input
+ * to a formula that fetches the current weather at that location, and the resulting object with
+ * weather info will be shown in the cell.
+ *
+ * A column format is just a wrapper around a formula defined in the {@link formulas} section
+ * of your pack definition. It tells Coda to execute that particular formula using the value
+ * of the cell as input.
+ *
+ * The formula referenced by a Format must have exactly one required parameter.
+ *
+ * You may optionally specify one or more {@link matchers}, which are regular expressions
+ * that can be matched against values that users paste into table cells, to determine if
+ * this Format is applicable to that value. This is a discovery mechanism to help users
+ * realize that there is a pack that may augment their experience of working with such values.
+ *
+ * For example, if you're building a Wikipedia pack, you may write a matcher regular expression
+ * that looks for Wikipedia article URLs, if you have a formula that can fetched structured data
+ * given an article URL. This would help users discover that there is a pack that can fetch
```

Comment:
Does the user need to have the pack installed for the matcher to run? Or are they run for all packs?

Reply:
In that case I'd remove or downplay the language about this being a discovery mechanism until that new regex matcher work is done.

Reply:
Makes sense, downplayed.